scripts/replace.js

```diff
@@ -164,6 +164,7 @@ const latestVersion = getRealVersion(latestMajorRelease);
 const from = [
   /@pulsar:version_number@/g,
   /@pulsar:version@/g,
+  /@pulsar:version_origin@/g,
```

Comment:
I'm afraid that such variable cannot resolve 2.6~2.7 versioned docs to 2.6.x and 2.7.x properly. But if we change only for new versions, it can be reasonable.

Reply:
yes, @momo-jun said only update the versions begin with `2.8.x`

Reply:
@tisonkun It's acceptable, IMO, because we no longer actively improve the docs of those versions.
I will update the occurrences of links to the reference site in 2.8.x docs and later versions as soon as this PR can be merged.

Reply:
Make sense. Although, this patch update for NEXT version only. @urfreespace would you update for 2.8.x, 2.9.x, 2.10.x and 2.11.x?

Reply:
yeah, you are right, I just keep the script update and removed the update for next version, and @momo-jun will make another PR to update the docs, PTAL